### ISSUE TITLE
a11y: alt text on actor profile avatar & banner

### ISSUE THREAD

**Issue by hugeblank** [bluesky-social/atproto#3187]
**Is your feature request related to a problem? Please describe.**

I'd like to be able to add alt text to my profile avatar and banner so I can provide a description, context and attribution.

**Describe the solution you'd like**

As it stands right now the avatar and banner defs are just URIs, I propose they should be something like an `app.bsky.embed.images#image` record.

**Describe alternatives you've considered**

Initially I thought about just throwing the alt text in with the profile view, but reusing existing lexicon definitions feels a lot smarter.

**Additional context**

At the moment only the avatar can be expanded in view on the app. The banner should be able to be expanded too, and then the alt text should appear somewhere below the image in the same way it does on post images:
![image](https://github.com/user-attachments/assets/6bf56e0d-ef31-4f09-b38d-e11b9dd9d66d)

**Comment by hugeblank** [comment]
Because I'm curious and want to contribute in the future I'd appreciate if someone peer reviewed my approach here. If I were to make a PR for this, I think all I'd have to do is in [`app/bsky/actor/defs.json`](https://github.com/bluesky-social/atproto/blob/main/lexicons/app/bsky/actor/defs.json), anywhere where there's an `"avatar"` or `"banner"` key, replace the value with:
```json
{ "type": "ref", "ref": "app.bsky.embed.images#image" }
```
